lms/models/region.py

```diff
@@ -27,17 +25,36 @@ class Regions:
 
     _AUTHORITY_MAP = {region.authority: region for region in ALL}
 
+    _current_region: Region = None
+
     @classmethod
-    def from_request(cls, request: Request) -> Region:
+    def get_region(cls) -> Region:
         """
-        Get a region object based on a request.
+        Get the region this app is running in.
 
-        :raises ValueError: If no valid region can be found.
+        :raises ValueError: If the active region has not been set
         """
-        authority = request.registry.settings["h_authority"]
+        if cls._current_region is None:
+            raise ValueError(
```

Comment:
I reckon this is enough to guard against any usage we might do that for some strange reason happen before `create_app`. Pretty unlikely.